src/status_im/ui/screens/chat/views.cljs

```diff
@@ -109,9 +110,48 @@
 
 (defn empty-chat-container
   []
-  [react/view style/empty-chat-container
-   [react/text {:style style/empty-chat-text}
-    (i18n/label :t/empty-chat-description)]])
+  (let [{:keys [color chat-id chat-name]}
+        @(re-frame/subscribe [:chats/current-chat])
+
+        link
+        (universal-links/generate-link :public-chat :external chat-id)
+
+        message
+        (i18n/label :t/share-public-chat-text {:link link})
+
+        public-chat-name-style
+        {:style {:font-size     24
+                 :font-weight   :bold
+                 :line-height   28
+                 :margin-bottom 10
+                 :color         colors/black}}
+
+        public-chat-icon-style
+        {:margin-bottom    25
+         :width            130
```

Comment:
width and height should be 120, bottom margin 24

Reply:
done!